enclave_manager/Dockerfile

```diff
@@ -293,6 +293,13 @@ RUN mkdir -p build \
   && cmake .. \
   && make
 
+WORKDIR /project/avalon/scripts
+
+ARG ENCLAVE_BINARY="../tc/sgx/trusted_worker_manager/enclave/build/lib/libavalon-wpe-enclave.signed.so"
```

Comment:
Since dockerfile is common to all type of workers, this section gets executed by all. So, can we move this to wpe enclave cmake file after the shared library is generated?

Reply:
ENCLAVE_BINARY -> WPE_ENCLAVE_BINARY
Or you can use `ENCLAVE_BINARY_PATH` to be generic. But in any case hard coding the wpe enclave binary name here looks bit ugly. Moving this logic to CmakeLists.txt seems a better option